File: Inventario/DataAccess/Models/MiscellaneousExpenses.cs
﻿using System;
using System.Collections.Generic;
using System.Text;

namespace Data
{
    public class MiscellaneousExpenses: Entity
    {
        public string Description { get; set; }
        public DateTime Date { get; set; }
        public decimal Value { get; set; }
        public string? Destination { get; set; }
        public bool IsFixed { get; set; }
    }
}

File: inventario_frontend/src/models/miscellaneousExpenses.ts
export class MiscellaneousExpensesForCreation {
  public description: string | undefined = undefined;
  public date: Date | undefined = undefined;
  public value: number | undefined = undefined;
  public destination?: string | undefined = undefined;
  public isFixed: boolean | undefined = undefined;
}

export class MiscellaneousExpenses extends MiscellaneousExpensesForCreation {
  public id!: string;
}
